lib/Report/TimeConnected.php

```diff
@@ -211,10 +211,32 @@ public function getSavedReportResults($json, $savedReport)
     /** @inheritdoc */
     public function getResults(SanitizerInterface $sanitizedParams)
     {
-        // Filter by displayId?
-        $displayIds = $this->getDisplayIdFilter($sanitizedParams);
-
-
+        $displayId = $sanitizedParams->getInt('displayId');
```

Comment:
I think we need to take a look at this together please - we're using `displayIds` as a filter again, which is not what we want to do.